### DISCUSSION TITLE
2021-08-18 Weekly Meeting 17 Agenda

### DISCUSSION THREAD

**Discussion by humbletim** [omigroup/omigroup#75] [Weekly Meetings]
Weekly Meeting 17 is happening on Wednesday 2021-08-18 from 9:00 AM PDT - 10:00 AM PDT

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR Discord](https://discord.gg/NJtT9grz5E).

## Agenda
   - Screenshot attendance, Welcome new members, review process for Submission and forming working groups
   - Working agreements / playbook follow-up
       - A quick statement from Jesse 
   - Website
       - A group is getting together this Sunday 8/22/21 at 10am PDT / 1pm EDT, to apply some of our recent changes to the existing site, and get a new iteration live. 
   - glTF Sub Group
       - OMI Audio Emitter Extension
       - Blender Tooling
       - Hubs -> OMI glTF Transform CLI
       - Robert created a brainstorming [list](https://github.com/omigroup/gltf-extensions/discussions/7) 
   - Discord Bot
       - progress update
   - World Exploration
       - schedule next outing
   - First Lightning Talk (prev Demo Days) event (~~August 26~~)
       - tentative event time moving to September
       - [sign-up form](https://forms.gle/EsXXnzUG2eaXQpxo7)
       - https://github.com/omigroup/OMI/issues/59
   - Screenshot end attendance (for those who arrived late) 

Add ons: 
- Virtual World URIs

Notes for todays session are available on [hackmd](https://hackmd.io/@mrmetaverse/omi-081821)

**Comment by humbletim** [comment]
quick follow-up on Virtual World URIs https://github.com/omigroup/OMI/discussions/23 discussion:
  - @jamesjackson's comment re: Open AR Cloud and the idea of considering AR locations similar to other virtualized world addresses
  - @jfenton's ideas around creating a neutral system app to help users actually consume virtual world URIs locally https://github.com/omigroup/OMI/discussions/23#discussioncomment-1188604
  - @technobaboo ideas around potential overlap with Stellar Mesh/Constellation
  - ... discuss relevant next steps (schedule a focused brainstorming meeting? is there sufficient interest to propose a subgroup?)

**Comment by robertlong** [comment]
I started a [list](https://github.com/omigroup/gltf-extensions/discussions/7) of possible glTF extensions for our community to work on next. I'd love more suggestions and input on the proposals I've made.

**Comment by mrmetaverse** [comment]
Re: Website 

A group is getting together this Sunday 8/22/21 at 10am PDT / 1pm EDT, to apply some of our recent changes to the existing site, and get a new iteration live. We can iterate on it from there, but probably have more than enough to work with for now.

**Comment by mrmetaverse** [comment]
Notes for todays session are started [on hackmd](https://hackmd.io/@mrmetaverse/omi-081821)
